packages/realm/.eslintrc.json

```diff
@@ -6,6 +6,7 @@
   ],
   "rules": {
     "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/no-duplicate-enum-values": "off",
```

Comment:
It would be nice if we had an eslint config file that was scoped to the generated source code, but .. this will work for now.